src/Morphir/IR/Literal/Codec.elm

```diff
@@ -110,6 +117,21 @@ decodeLiteral =
                                     (\str -> Decode.succeed <| dec str)
```

Comment:
I came across this 
`Decimal.fromString v |> Maybe.withDefault (Decimal.fromInt 0)` 
while adding in the `LocalDateLiteral`.

Defaulting the value to 0 doesn't seem right. 
I think we could handle this better and fail rather.

Reply:
Agreed. Nice catch! Can you please change it to fail as you said? Thanks